setup.py

```diff
@@ -96,9 +126,9 @@
     extras["flax"] = [] # jax is not supported on windows
 else:
     extras["retrieval"] = ["faiss-cpu", "datasets"]
-    extras["flax"] = ["jaxlib==0.1.55", "jax>=0.2.0", "flax==0.2.2"]
+    extras["flax"] = deps_list("jax", "jaxlib", "flax")
 
-extras["tokenizers"] = ["tokenizers==0.9.4"]
+extras["tokenizers"] = deps_list("tokenizers")
```

Comment:
@sgugger, are you happy with setup.py's part where the extras are filled out to specific versions, before I proceed with the rest of the packages?

Reply:
I don't want to re-do it - so want to know whether this is good or you'd rather have a different way...

Reply:
This LGTM.